lib/core/utils.js

```diff
@@ -1105,9 +1105,9 @@ RESUtils.niceDateDiff = function(origdate, newdate) {
 	var aday = origdate.getUTCDate();
 	var ayear = origdate.getUTCFullYear();
 
-	var tyear = newdate.getUTCFullYear();
 	var tmonth = newdate.getUTCMonth() + 1;
 	var tday = newdate.getUTCDate();
+	var tyear = newdate.getUTCFullYear();
```

Comment:
Oh because I wanted to match the order of the original date variables. I guess it's a little bit nitpicky, but I saw no reason why the variables wouldn't have the same order in the first place, so I just shifted the line.